cleanlab/datalab/datalab.py

```diff
@@ -0,0 +1,682 @@
+
+            Examples
+            --------
+
+            Suppose you want to detect label issues. Just pass a dictionary with the key "label" and an empty dictionary as the value.
+
+            .. code-block:: python
+
+                issue_types = {"label": {}}
+
+
+            For more control, you can pass keyword arguments to the issue manager that handles the label issues.
+            For example, if you want to pass the keyword argument "clean_learning_kwargs"
+            to the constructor of the LabelIssueManager, you would pass:
+
+
+            .. code-block:: python
+
+                issue_types = {
+                    "label": {
+                        "clean_learning_kwargs": {
+                            "prune_method": "prune_by_noise_rate",
+                        },
+                    },
+                }
+        """
+
+        if issue_types is not None and not issue_types:
+            warnings.warn(
+                "No issue types were specified. " "No issues will be found in the dataset."
+            )
+            return None
+
+        issue_types_copy = self.get_available_issue_types(
+            pred_probs=pred_probs,
+            features=features,
+            knn_graph=knn_graph,
+            model=model,
+            issue_types=issue_types,
+        )
+
+        new_issue_managers = [
+            factory(datalab=self, **issue_types_copy.get(factory.issue_name, {}))
+            for factory in _IssueManagerFactory.from_list(list(issue_types_copy.keys()))
+        ]
+
+        if not new_issue_managers:
+            no_args_passed = all(arg is None for arg in [pred_probs, features, knn_graph, model])
+            if no_args_passed:
+                warnings.warn("No arguments were passed to find_issues.")
+            warnings.warn("No issue check performed.")
+            return None
+
+        failed_managers = []
+        for issue_manager, arg_dict in zip(new_issue_managers, issue_types_copy.values()):
+            try:
+                if self.verbosity:
+                    print(f"Finding {issue_manager.issue_name} issues ...")
+                issue_manager.find_issues(**arg_dict)
+                self.data_issues.collect_statistics_from_issue_manager(issue_manager)
+                self.data_issues._collect_results_from_issue_manager(issue_manager)
+            except Exception as e:
+                print(f"Error in {issue_manager.issue_name}: {e}")
+                failed_managers.append(issue_manager)
+
+        if self.verbosity:
+            print(
+                f"Audit complete. {self.issue_summary['num_issues'].sum()} issues found in the dataset."
+            )
+        if failed_managers:
+            print(f"Failed to check for these issue types: {failed_managers}")
+
+        self.data_issues.set_health_score()
+
+    def get_available_issue_types(self, **kwargs):
+        """Returns a dictionary of issue types that can be used in the find_issues method."""
```

Comment:
```suggestion
        """Returns a dictionary of possible issue types that can be used in the ``find_issues`` method."""
```

Reply:
proper link to the method would be better, sorry I don't recall optimal way to do it for instance methods

Reply:
I think it is:
```
:py:meth:`Datalab.find_issues <cleanlab.datalab.datalab.Datalab.find_issues>`
```